src/component/toolbox/feature/DataView.ts

```diff
@@ -324,6 +324,9 @@ class DataView extends ToolboxFeature<ToolboxDataViewFeatureOption> {
             api.dispatchAction({
                 type: 'hideTip'
             });
+            api.dispatchAction({
+                type: 'openDataView'
```

Comment:
I think `showDataView` and `hideDataView` should be a better name to align with `showTip` and `hideTip`.